docs/source/workflows/functions/text-to-sql.md

```
@@ -0,0 +1,521 @@
+- **`false`** (default): Uses manually defined training data from `training_db_schema.py` (`VANNA_TRAINING_DDL`, `VANNA_TRAINING_EXAMPLES`, `VANNA_TRAINING_DOCUMENTATION`). This gives you full control over the training data quality.
+
+### Database Configuration
+
+**Databricks:**
+```yaml
+database_type: databricks
+connection_url: "databricks://token:${DB_TOKEN}@${DB_HOST}:443/default?http_path=${HTTP_PATH}&catalog=main&schema=default"
+```
+
+**Connection URL Format:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)